src/commands/start_api.py

```diff
@@ -155,6 +162,13 @@
     prompt='Enter comma separated list of API endpoints for consensus nodes',
     help='Comma separated list of API endpoints for consensus nodes.',
 )
+@click.option(
+    '--graph-api-url',
```

Comment:
Would be great to support multiple graph nodes as an argument similarly to execution and consensus endpoints. Also, naming is inconsistent, lets keep the same: --graph-endpoints